.gitignore

```diff
@@ -123,3 +123,4 @@ StressLog.txt
 *.csv
 *.gcdump
 *.binlog
+/out/build/x64-Debug
```

Comment:
What generates this directory? It it really necessary?

Reply:
This I think VS Code did that. That is not needed. That can be removed. I will do that.